classroom.rb

```diff
@@ -0,0 +1,15 @@
+# Class Classroom
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @add_student << student
+    student.classroom = self
+  end
```

Comment:
- [ ] Let's push the student received as an argument to the `@students` instance variable which is an array defined in the initialize method. Right now you're pushing to `@add_student` instance variable which does not exist. See suggestion below;

```suggestion
  def add_student(student)
    @students << student
    student.classroom = self
  end
```